_sass/content.scss

```
@@ -112,3 +112,56 @@
     font-weight: 500;
   }
 }
+
+banner {
+  width: 100%
```

```suggestion
  width: 100%;
```

Sorry about that 😭 , @shaynacohen